scene/resources/world_2d.cpp

```diff
@@ -387,7 +386,7 @@ World2D::World2D() {
 
 	//set space2D to be more friendly with pixels than meters, by adjusting some constants
 	Physics2DServer::get_singleton()->space_set_active(space, true);
-	Physics2DServer::get_singleton()->area_set_param(space, Physics2DServer::AREA_PARAM_GRAVITY, GLOBAL_DEF("physics/2d/default_gravity", 98));
+	Physics2DServer::get_singleton()->area_set_param(space, Physics2DServer::AREA_PARAM_GRAVITY, GLOBAL_DEF("physics/2d/default_gravity", 9.8));
```

Comment:
As long as the value is multiplied by ten where it's used, the behavior remains the same and the value exposed in the project settings is the expected one, so imho it's still a good change. What do you think?